handler.py

```diff
@@ -57,23 +58,26 @@ def delete_message(group_id, message_id, token):
 
 
 def kick_user(group_id, user_id, token):
-    membership_id, _ = get_membership_id_and_admin_status(group_id, user_id, token)
-    remove_member(group_id, membership_id, token)
+    membership_id = get_membership_id(group_id, user_id, token)
+    if membership_id:
+        return remove_member(group_id, membership_id, token)
+    return False
+
 
 
 
 def receive(event, context):
     message = json.loads(event['body'])
-
     bot_id = message['bot_id']
+    
     for phrase in FLAGGED_PHRASES:
         if phrase in message['text'].lower():
-            membership_id, is_admin = get_membership_id_and_admin_status(message['group_id'], message['user_id'], message['token'])
-            # Proceed with actions only if the user is not an admin
-            if not is_admin:
-                kick_user(message['group_id'], message['user_id'], message['token'])
+            # Attempt to kick the user and check if it was successful
```

Comment:
Here is where I switched the logic to check the HTTP status of kick user instead of some admin flag that doesn't exist.